particles/canonical.arcs

```diff
@@ -20,4 +20,6 @@ import 'Tutorial/Kotlin/Tutorial.arcs'
 import 'TicTacToe/TicTacToe.arcs'
 // Other
 import 'Native/native.arcs'
+// Test
+import 'tests/WasmIntegrationTest.arcs'
```

Comment:
We may want to consider locating this under `src/tests` - there's no real benefit to having it under `particles` other than making it easier to import here (although you could argue there's some utility in being able to manually invoke the integration test from the shell UI).

Not necessary for this CL, in any case.